src/v/pandaproxy/schema_registry/seq_writer.cc

```diff
@@ -228,6 +228,41 @@ ss::future<bool> seq_writer::write_config(
     });
 }
 
+ss::future<std::optional<bool>> seq_writer::do_delete_config(
+  std::optional<subject> sub, model::offset write_at, seq_writer& seq) {
+    vlog(plog.debug, "delete config sub={} offset={}", sub, write_at);
+
+    try {
+        if (sub.has_value()) {
+            (void)co_await seq._store.get_compatibility(
```

Comment:
clarification: adding void here is necessary to throw the return value away?

Reply:
Yeah I think that's right. Cribbed from `seq_writer::do_delete_config`

Reply:
I don't think it's needed.